crates/admin/protobuf/cluster_ctrl_svc.proto

```diff
@@ -16,6 +16,8 @@ import "google/protobuf/empty.proto";
 package restate.cluster_ctrl;
 
 service ClusterCtrlSvc {
+  rpc GetVersion(google.protobuf.Empty) returns (restate.common.ApiVersion);
```

Comment:
This is looking great!

If we choose to make this gRPC as the "control facade" that's the primary access point for `restatectl`, maybe we don't need to add version info in other places - we can establish individual node's versions within the cluster, and use this operation from `restatectl` to establish client/server compatibility.

Reply:
I agree using the metadata would be simpler of course. It was on the table, the only reason(s) I didn't do it like that was:
- Obscure, it's not intuitive how to get the version without looking it up in docs.
- If you want only to get the version (maybe to show somewhere) you will have to still make one call, and discard its output. 
- Almost nobody will call GetVersion unless they detect a compatibility issue (e.g received a invalid_data response from a call)

That being said, I ended up only relying on the `server_version` that I added to the `IdentResposne` which means we can definitely drop the `GetVersion` completely and only introduce when it's needed